### ISSUE TITLE
Lutris searches for libasound_module_pcm_pulse.so at the wrong location

### ISSUE THREAD

**Issue by jeffshee** [lutris/lutris#1007]
Problem: Can't play any games without disable Lutris Runtime

OS: Fedora 28

Error message:
ALSA lib dlmisc.c:272:(snd1_dlobj_cache_get) Cannot open shared library /usr/lib/x86_64-linux-gnu/alsa-lib/libasound_module_pcm_pulse.so

Is actually located at:
/usr/lib/alsa-lib/libasound_module_pcm_pulse.so
/usr/lib64/alsa-lib/libasound_module_pcm_pulse.so

**Comment by Isakku** [comment]
Confirmed! I have this problem! StarCraft Remastered has sound in one of my Linux Mint (Cinnamon), and lacks sound in another of my Linux Mint (KDE) because of this!
I have opened a thread at the forums,
https://forums.lutris.net/t/starcraft-remastered-no-sound/2931/9
 but I found this issue right now.

From game log:

`ALSA lib conf.c:3357:(snd_config_hooks_call) Cannot open shared library libasound_module_conf_pulse.so`

I do have the libraries, I even tried this:
https://askubuntu.com/questions/542984/unable-to-open-shared-libasound-library-despite-file-existing
```
cd /usr/lib/x86_64-linux-gnu/
sudo ln -s alsa-lib/libasound_module_conf_pulse.so libasound_module_conf_pulse.so
```
To no avail. 
Now the curious thing is that not all games lack sound, only StarCraft Remastered does, other games do display that in the game log (Heroes Of The Storm) but there is no problem with the sound, and it only happens in the Linux MInt KDE, but not on Linux Mint Cinnamon that doesn't display that in the debug (It is the same hardware, I'm dual booting)
So, this bug does happen, it exists indeed, but I can bet that tracking it will be tricky because it only happens in a certain situation, wich is the case of my Mint KDE box setup.

**Comment by strycore** [comment]
I'm almost sure this is fixed by the runtime changes in the upcoming release, if not we'll have to investigate more.

**Comment by Isakku** [comment]
@strycore thanks a lot! So... will that new release come to Linux Mint 18.3 as soon as you release it? Or are we condemned by the package base to upgrade to Linux MInt 19? I hope we are not. I really wish there were a flatpaked version, https://forums.lutris.net/t/lutris-for-flatpak-in-progress/1257/6
using Flatpak tech to ensure that Lutris has all the dependencies and also using flatpak tech to ensure all runners have all their requirements met (that's what Winepak aims to do for Wine Windows apps) and finally using Flatpak to avoid this kind of issues,

**Comment by strycore** [comment]
I just know that Mint is based on Ubuntu and that we have packages for Ubuntu 16.04 and 18.04, so that should get you covered

**Comment by strycore** [comment]
Fixed in 0.4.19

**Comment by Isakku** [comment]
This made things worse for me. Now nothing works! Blizzard games don't even launch anymore after upgradign to 4.19!
